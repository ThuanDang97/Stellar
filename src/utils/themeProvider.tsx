import { ThemeProvider } from 'styled-components'
import { render } from '@testing-library/react'
import { theme } from '@themes/index'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>)
}
