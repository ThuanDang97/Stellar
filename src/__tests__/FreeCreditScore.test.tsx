import React from 'react'

// Utils
import { renderWithTheme } from '@utils/themeProvider'

// Page
import FreeCreditScore from '@pages/free-credit-score'

describe('Free credit score Page', () => {
  it('Should be renders free credit score page ', () => {
    const { asFragment } = renderWithTheme(<FreeCreditScore />)

    expect(asFragment()).toMatchSnapshot()
  })
})
