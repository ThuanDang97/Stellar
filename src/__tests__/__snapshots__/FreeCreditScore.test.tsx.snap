// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Free credit score Page Should be renders free credit score page  1`] = `
<DocumentFragment>
  <div
    class="ViewStyled-sc-rqe5sa-0 ViewStyled__WrapperContent-sc-rqe5sa-6 gOHbQa gVWyht"
    display="flex"
  >
    <h1
      class="TextStyled-sc-9xia41-0 TitleStyled-sc-t58pjn-0 TitleStyled__TitlePageStyled-sc-t58pjn-1 gwtqfM ONSni ilgUUT"
      color="#242A30"
      font-family="AdobeCleanExtraBold"
      font-size="36"
      letter-spacing="2"
    >
      Get your free credit score
    </h1>
    <span
      class="TextStyled-sc-9xia41-0 ccHUFb"
      letter-spacing="0.9"
    >
      This will not impact your score and only takes a minute to complete.
    </span>
    <div
      class="ViewStyled-sc-rqe5sa-0 cFluyW"
      height="450"
      width="inherit"
    >
      <span
        style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
      >
        <img
          alt="Jumping man image"
          data-nimg="fill"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
        />
        <noscript />
      </span>
    </div>
    <a
      class="LinkStyled__ButtonLinkStyled-sc-626068-1 cWbYJP"
      data-testid="link"
      href="/confirm-identity"
      width="320"
    >
      Get My Score
    </a>
  </div>
</DocumentFragment>
`;
